com.unity.netcode.gameobjects/Runtime/NetworkVariable/ClientNetworkVariable.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+
+namespace Unity.Netcode
+{
+    /// <summary>
+    /// A ClientNetworkVariable is special in that:
+    ///  - only the owner of the variable can write to it
+    ///  - not even the server can write to it
```

Comment:
> not even the server can write to it
I might have missed this with Owernship, but to me this would be covered by the IsOwner check no? IsOwner checks for 
`NetworkManager != null && OwnerClientId == NetworkManager.LocalClientId;`
This would be false for the server?